### PR TITLE
Updated German locale

### PR DESCRIPTION
**Pull request by invliD**

### CODE REVIEW

File: _1327/documents/locale/de/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+msgid "Your Title..."
+msgstr "Titel"
+
+#: templates/editor.html:26
+msgid "Your Text..."
+msgstr "Text"
+
+#: templates/editor.html:32
+msgid "Created document"
+msgstr "Dokumennt erstellt"
```
Comment (invliD) [inline _1327/documents/locale/de/LC_MESSAGES/django.po]: Argh!

Comment (Bartzi) [inline _1327/documents/locale/de/LC_MESSAGES/django.po]: ohoh....

File: _1327/documents/templates/documents/admin_app_name.html [hunk 2]
```diff
@@ -0,0 +1,2 @@
+{% load i18n %}
+{% trans "Documents" %}
```
Comment (Bartzi) [inline _1327/documents/templates/documents/admin_app_name.html]: Could you explain us why we need this file? Looks interesting :smile:

**Comment by Bartzi** [comment]
looks good  now :smile:  :+1:

File: _1327/documents/templates/documents/admin_app_name.html (hunk 2 above)
Comment (invliD) [inline _1327/documents/templates/documents/admin_app_name.html]: This file is rendered as the app name in Django admin, allowing localization of the app's name.

**Pull request merged**